src/assets/i18n/pt-PT.json5

```diff
@@ -7094,6 +7130,30 @@
   // "workspace-item.view.title": "Workspace View",
   "workspace-item.view.title": "Vista das tarefas",
 
+  // "workspace-item.delete.breadcrumbs": "Workspace Delete",
+  "workspace-item.delete.breadcrumbs": "Apagar da área de trabalho",
+
+  // "workspace-item.delete.header": "Delete workspace item",
+  "workspace-item.delete.header": "Apagar item da área de trabalho",
+
+  // "workspace-item.delete.button.confirm": "Delete",
+  "workspace-item.delete.button.confirm": "Apagar",
+
+  // "workspace-item.delete.button.cancel": "Cancel",
+  "workspace-item.delete.button.cancel": "Cancelar",
+
+  // "workspace-item.delete.notification.success.title": "Deleted",
+  "workspace-item.delete.notification.success.title": "Apagado",
+
+  // "workspace-item.delete.title": "This workspace item was successfully deleted",
+  "workspace-item.delete.title": "Este item foi removido da área de trabalho com êxito",
```

Comment:
Can you consider:
`Este item foi removido, com êxito, da área de trabalho`

Reply:
Revised. Thanks!